### ISSUE TITLE
1.16.4: Combining Conditions doesn't work in formats

### ISSUE THREAD

**Issue by JessePesse123** [kangarko/ChatControl#392]
Hello, I think there is a bug regarding to combining conditions in formats. 

**What I want to do:** 
I want to make two format parts in my main chat:
- Pvp Status:
```
pvpstatus:
    Message: '&4☠ '
    Receiver_Permission: chatcontrol.pvpstatus
    Sender_Condition: '{pvpmanager_pvp_status}'
    Hover:
    - '&cKlik hier om alle spelers met pvp te zien'
    Suggest_Command: /pvpl
```
- Mcmmoparty:
```
mcmmo-party:
    Message: '&2[&f{mcmmo_party_name}&2] '
    Receiver_Permission: chatcontrol.mcmmoparty
    Sender_Condition: '{mcmmo_in_party}'
    Hover:
    - '&8*_____*&f {mcmmo_party_name} &8*_____* &f'
    - '&6Leden&b: &f{mcmmo_party_size}'
    - '&6Leider&b: &f{mcmmo_party_leader}'
    - '&7&oKlik voor jouw party info'
    Run_Command: /party info
```
However, when I create these parts, the Discord stops working, resulting in these errors _(view: error1)_
I must add` Sender_Condtion: '{sender_is_player}'` So i need to combine them. I contacted you and tried your method:
![image](https://user-images.githubusercontent.com/66204768/103176869-a74ac200-4875-11eb-91e7-17337a5c5eef.png)

But it didn't work, Then Banana and i tried alot of methods, which all had their own kind of errors. :

1. Banana tried creating a Custom variable {player_in_party}, which checks if the sender is a player and if he's in a party. This Sadly didn't work: _(view: Player-in-party.yml and error2)_

2. We tried doing this: `Sender_Condition: '{sender_is_player} && {mcmmo_in_party}'`, this worked in chat, but not from discord.
_(view: error3)_

3. We tried another method: `'{mcmmo_in_party} && {sender_is_discord} == false' ` but this resulted in another error _(view: error4)_

4.  We tried a new method: `'!{sender_is_discord} && {mcmmo_in_party}'` , this didn't work either _(view: error5)_ 

So all things concluded, we tried alot of methods, but none of them worked. We don't know what we did wrong and thereby think it is an issue in the plugin itself. This would all be alot easier if we could specify a `discord.yml` format file for messages sent through discord, that way we don't have to bother with variables etc. that way you can get a cleaner` chat.yml `aswell

I also provided my chatformat. Hope this is enough, feel free to contact me if you need more info

Things to do to reproduce: try any of our methods. 

Minecraft Version: 1.16.4
MySQL: false
BungeeCord: false

All files are here: [https://ufile.io/f/s3na5](url)

**Comment by kangarko** [comment]
@ElBananaa can you please have a look? Many thanks!

**Comment by ElBananaa** [comment]
> @ElBananaa can you please have a look? Many thanks!

@kangarko As said on discord, we already tried to fix this issue with its author for something like an hour and a half before I asked him to finally open a ticket here. To summarize: Using 2 conditions in a format works ingame ingame, but using 2 conditions for the same format with discordSRV relay enable doesn't work, the 2nd placeholder used isn't replaced, resulting in an error. I gave you more details in PM on discord.
